Resources/Prototypes/Entities/Clothing/Hands/base_clothinghands.yml

```diff
@@ -22,6 +22,7 @@
   - type: Tag
     tags:
     - ClothMade
+    - Recyclable
```

Comment:
Add it to ClothingHandsButcherable instead. Otherwise it causes some gloves to be recyclable, but have no materialComposition.